tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/query/QueryTypeNarrowingTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Reply:
This change was introduced due to renaming the file. Hence, should be fine to keep it